jdwp-shellifier.py

```diff
@@ -419,64 +422,64 @@ def parse_event_breakpoint(self, buf, eventId):
 
 
 def runtime_exec(jdwp, args):
-    print ("[+] Targeting '%s:%d'" % (args.target, args.port))
-    print ("[+] Reading settings for '%s'" % jdwp.version)
+    print(("[+] Targeting '%s:%d'" % (args.target, args.port)))
```

Comment:
Double parenthesis, also if we jump to Py 3.9 might as well use more goodies: here f-strings
```suggestion
    print(f"[+] Targeting '{args.target}:{args.port}'")
```